packages/client/src/forms/configuration/default.ts

```diff
@@ -4332,6 +4337,1326 @@ export const registerForms: IDefaultRegisterForms = {
+                  },
+                  query: {
+                    operation: 'nameToFieldTransformer',
+                    parameters: ['en', 'witnessOneFirstName']
+                  }
+                }
+              },
+              {
+                name: 'witnessOneFamilyNameEng',
+                previewGroup: 'witnessOneNameInEnglish',
+                type: 'TEXT',
+                label: formMessageDescriptors.familyName,
+                maxLength: 32,
+                required: true,
+                initialValue: '',
+                validate: [
+                  {
+                    operation: 'englishOnlyNameFormat'
+                  }
+                ],
+                mapping: {
+                  template: {
+                    fieldName: 'witnessOneFamilyNameEng',
+                    operation: 'nameToFieldTransformer',
+                    parameters: ['en', 'witnessOneFamilyName']
+                  },
+                  mutation: {
+                    operation: 'fieldToNameTransformer',
+                    parameters: ['en', 'witnessOneFamilyName']
+                  },
+                  query: {
+                    operation: 'nameToFieldTransformer',
+                    parameters: ['en', 'witnessOneFamilyName']
+                  }
+                }
+              },
+              {
+                name: 'witnessOneRelationship',
+                customisable: false,
+                type: 'SELECT_WITH_OPTIONS',
+                label: formMessageDescriptors.relationshipToSpouses,
+                required: true,
+                initialValue: '',
+                validate: [],
+                placeholder: formMessageDescriptors.formSelectPlaceholder,
+                mapping: {
+                  template: {
+                    fieldName: 'witnessTwoRelationship',
+                    operation: 'selectTransformer'
+                  }
+                },
+                options: [
+                  {
+                    value: 'headOfGroomFamily',
+                    label: formMessageDescriptors.headOfGroomFamily
+                  },
+                  {
+                    value: 'headOfBrideFamily',
+                    label: formMessageDescriptors.headOfBrideFamily
+                  },
+                  {
+                    value: 'other',
+                    label: formMessageDescriptors.other
+                  }
+                ]
+              },
+              {
+                name: 'witnessOneOtherRelationship',
+                type: 'TEXT',
+                label: formMessageDescriptors.other,
+                maxLength: 32,
+                required: true,
+                initialValue: '',
+                validate: [],
+                conditionals: [
+                  {
+                    action: 'hide',
+                    expression: '(values.witnessOneRelationship!="other")'
+                  }
+                ],
+                mapping: {}
```

Comment:
We would either need to provide mapping functions or not provide mapping at all instead of just an empty object